### ISSUE TITLE
No macOS package for vimix 0.8.1

### ISSUE THREAD

**Issue by cpkcpkcpk** [brunoherbelin/vimix#108]
Hey, just noticed that there is no macOS release package in the assets section for 0.8.1 like the previous versions.

There's probably a reason for that, but just pointing it out in case there is meant to be!

Some of the new features look very useful, and can build if necessary but would prefer not to :D

**Comment by brunoherbelin** [comment]
Indeed, the reason is the complexity of building and signing OSX packages without programming using XCode and Apple development tools (I am not an Apple developer, nor want to become one...). 

I welcome the help of OSX programmers who know how to setup CMakeLists instructions for CPack 
(i.e. replace old code between lines 185 and 280 of the [CMakeLists.txt](https://github.com/brunoherbelin/vimix/blob/master/src/CMakeLists.txt) )

**Comment by brunoherbelin** [comment]
I also considered setting up a package for [Homebrew](https://brew.sh/), but either the program is built as a pure brew package and has to be launched from command line (which is not very nice), or the program anyways has to be compiled as a normal OSX Bundle and only envelopped into a homebrew cask (which is back to the original problem of compiling and signing the OSX Bundle with CMake).

I am confident there are programmers who know how to do this (e.g. the people packaging OBS studio) but vimix community is much smaller ...

**Comment by brunoherbelin** [comment]
Sorry for the wait... version 0.8.2 now has an intaller
https://github.com/brunoherbelin/vimix/releases/tag/0.8.2b
